<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-E42ICP36" failures="0" tests="3" name="extent" time="0.057" errors="0" timestamp="2022-01-24T17:22:06 IST">
  <testcase classname="extent.Report" name="@BeforeSuite openBrowser" time="0.058">
    <failure type="java.lang.NoClassDefFoundError" message="io/reactivex/rxjava3/subjects/PublishSubject">
      <![CDATA[java.lang.NoClassDefFoundError: io/reactivex/rxjava3/subjects/PublishSubject
at com.aventstack.extentreports.ReactiveSubject.<init>(ReactiveSubject.java:28)
at com.aventstack.extentreports.AbstractProcessor.<init>(AbstractProcessor.java:22)
at com.aventstack.extentreports.ExtentReports.<init>(ExtentReports.java:55)
at extent.Report.openBrowser(Report.java:64)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:317)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.lang.ClassNotFoundException: io.reactivex.rxjava3.subjects.PublishSubject
at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641)
at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:188)
at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:520)
... 23 more
]]>
    </failure>
  </testcase> <!-- @BeforeSuite openBrowser -->
  <testcase classname="extent.Report" name="openBing" time="0.0">
    <skipped/>
  </testcase> <!-- openBing -->
  <testcase classname="extent.Report" name="openGoogle" time="0.0">
    <skipped/>
  </testcase> <!-- openGoogle -->
  <testcase classname="extent.Report" name="openYahoo" time="0.0">
    <skipped/>
  </testcase> <!-- openYahoo -->
</testsuite> <!-- extent -->
